README.md

```diff
@@ -67,6 +70,18 @@ Your Swagger definition lives here:
 http://localhost:1071/uji/api/v1/swagger.json
```

Comment:
Typo in URL? Should be:
http://localhost:1071/fuji/api/v1/ui/

Reply:
@vemonet Could you please check if this is a typo? See @wilkos-dans comment above

Reply:
@vemonet please forget my last comment